integration-cli/docker_api_exec_test.go

```diff
@@ -19,12 +20,11 @@ func (s *DockerSuite) TestExecApiCreateNoCmd(c *check.C) {
 	dockerCmd(c, "run", "-d", "-t", "--name", name, "busybox", "/bin/sh")
 
 	status, body, err := sockRequest("POST", fmt.Sprintf("/containers/%s/exec", name), map[string]interface{}{"Cmd": nil})
-	c.Assert(err, check.IsNil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
 
-	if !bytes.Contains(body, []byte("No exec command specified")) {
-		c.Fatalf("Expected message when creating exec command with no Cmd specified")
-	}
+	comment := check.Commentf("Expected message when creating exec command with no Cmd specified")
+	c.Assert(body, checker.Contains, "No exec command specified", comment)
```

Comment:
tests are failing because `No exec command specified` isn't a `[]byte{}` here, I think you either cast `body` to a `string` or the message to a `[]byte`